LaunchClient.bat

```diff
@@ -1,3 +1,3 @@
 @echo off
 call "%~dp0ProjectPaths.bat"
-"%UNREAL_HOME%\Engine\Binaries\Win64\UE4Editor.exe" "%~dp0%PROJECT_PATH%\%GAME_NAME%.uproject" -game -log -workerType UnrealClient -stdout -nowrite -unattended -nologtimes -nopause -noin -messaging -NoVerifyGC -windowed -ResX=800 -ResY=600
+"%UNREAL_HOME%\Engine\Binaries\Win64\UE4Editor.exe" "%~dp0%PROJECT_PATH%\%GAME_NAME%.uproject" 127.0.0.1 -game -log -workerType UnrealClient -stdout -nowrite -unattended -nologtimes -nopause -noin -messaging -NoVerifyGC -windowed -ResX=800 -ResY=600
```

Comment:
Don't think this is actually necessary, it should connect to localhost on startup unless you've passed something else in for `+receptionistHost`

Reply:
Yep, this has changed with the recent Server Travel PR, slightly jumping the gun with this fix.